bot/middleware/commands.ts

```diff
@@ -12,7 +14,7 @@ const commandArgs = () => (ctx, next) => {
           command = match[1];
         }
         let next_arg = ['', '', match[2]];
-        while ((next_arg = re_next_arg.exec(next_arg[2]))) {
+        while ((next_arg = re_next_arg.exec(next_arg[2])!)) {
```

Comment:
por qué cambiar la condición de while a not?

Reply:
@grunch hey Francisco, @webwarrior-ws doesn't speak Spanish

Reply:
sorry, why to change the while condition to not?

Reply:
I did not change condition. `!` in this case is not prefix negation operator, but postfix "assume not null" operator, which doesn't change code semantics and is only needed for TypeScript to not complain.